label_studio/tests/suits.py

```diff
@@ -0,0 +1,97 @@
+import os
+
+
+samples_path = os.path.join(os.path.dirname(__file__), '../','static/samples/')
+
+
+class Scenarios:
+    """
+        wrapper class for scenarios
+
+        each scenario should be a dict
+        with the same name
+
+        xxx_scenario = {
+            xxx_scenario: {
+                'actions':[],
+                'data1': 1,
+                'dataN': N,
+            }
+        }
+    """
+
+    test_common_scenario = {
```

Comment:
may be in this way naming test suites from dict keys is redundant  - it's possible to use Scenario attribute name itself as a test suite name